client/verta/verta/client.py

```diff
@@ -207,6 +207,8 @@ def is_unauthorized(response): return response.status_code == 401
         self._ctx = _Context(self._conn, self._conf)
         self._workspace = None
 
+        self.operations = MonitoringClient(self)
```

Comment:
Similar to https://github.com/VertaAI/modeldb/pull/2077/files#r607208703, I'd prefer this to be a `@property`, but that might just be what I'm used to.